### DISCUSSION TITLE
Is there an option to set default FS properties?

### DISCUSSION THREAD

**Discussion by druesendieb** [topolvm/topolvm#744] [Q&A] [answered]
Hi,

I am wondering if there is the possibility on the csi-drivers side to set default FS properties for volumes/storage classes? 

I didn't find a valid option on the kubernetes side, only [a closed issue](https://github.com/kubernetes/kubernetes/issues/82372). This hints to me, that this (formatting the fs) is something k8s itself takes care of but I wanted to ask you nonetheless. 

I have searched through the repository but wasn't lucky. I found the `additionalParameter` field in the storage class, but i haven't found file system operations related parameters (and with the find on the k8s side I guess there are none). 

In our case we're looking specifically at the reserved block count for ext4, which is 5% and should be decreased by default.

thanks in advance 🙇

**Comment by daichimukai** [comment, marked as answer]
Unfortunately, there is no way to set filesystem property for volumes or storage classes as far as I know. I agree that the reserved block count for ext4 might be excessive for somewhat big volumes, but I have no idea to address this issue :(

**Reply by druesendieb** [reply]
Okay, so I have to come up with a scripted solution. Thanks a lot for your answer!
